### PR TITLE
#535 possibility to send mails to different status groups

### PR DESCRIPTION
**Pull request by denys281**
Resolves [possibility to send mails to different status groups](https://github.com/pivorakmeetup/pivorak-web-app/issues/535)

### Description

On confirmation stage ads posibility to send email to applied and pending users
On registration stage ads posibility to send email to pending and approved  users

Move email buttons to separate partial
Ads specs for new finders
Ads traits for status to visit requests factories

### How to test instructions

**1**

1. Go to admin panel
2. Go to event
3. Update event status to registration stage

**Expected result:** You should see two buttons: Send email to pending users and Send email to approved users

**2**

1. Go to admin panel
2. Go to event
3. Update event status to registration stage
4. Click to **Send email to pending users** button

**Expected result:** You should be redirected to page where in the recipients list should be all users with pending status

**3**

1. Go to admin panel
2. Go to event
3. Update event status to registration stage
4. Click to **Send email to approved users** button

**Expected result:** You should be redirected to page where in the recipients list should be all users with approved status

**4**

1. Go to admin panel
2. Go to event
3. Update event status to confirmation stage

**Expected result:** You should see two buttons: Send email to pending users and Send email to applied users

**5**

1. Go to admin panel
2. Go to event
3. Update event status to registration stage
4. Click to **Send email to pending users** button

**Expected result:** You should be redirected to page where in the recipients list should be all users with pending status

**6**

1. Go to admin panel
2. Go to event
3. Update event status to registration stage
4. Click to **Send email to applied users** button

**Expected result:** You should be redirected to page where in the recipients list should be all users that applied to event (no matter with what status they are)

### Pre-merge checklist
 
- [ ] The PR relates to a single subject with a clear title and description
- [ ] Verify that feature branch is up-to-date with `development` (if not - rebase it). 

### Screenshots

confirmation
registration

### CODE REVIEW

**Review by denys-medynskyi — DISMISSED** [review]
Great! 👍 
Please notify Oksana and Anya how to use this feature. 
Please also consult with them which feature should be done next if you haven't done it before.

File: app/views/admin/visit_requests/_emails.slim [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.twelve.wide.column
+  = send_email_to_recipient_ids_link(visitors_ids)
+  = send_email_to_attendees_link(confirmed_ids)
+  = send_email_to_applied_users_link(applied_ids) if event.confirmation?
```
Comment (floor114) [inline app/views/admin/visit_requests/_emails.slim]: I would move these `if's` to helper methods like we have in other places.

**Review by floor114 — DISMISSED** [review]

**Review by mamantoha — APPROVED** [review]

**Comment by denys281** [comment]
@denysmedynskyi Okay. Could you review again and merge?

File: app/views/admin/visit_requests/_emails.slim (hunk 1 above)
Comment (denys281) [reply app/views/admin/visit_requests/_emails.slim]: @floor114  please, re-check

**Review by floor114 — APPROVED** [review]

**Review by denys-medynskyi — APPROVED** [review]

**Comment by denys-medynskyi** [comment]
@denys281 sorry, missed the notification from you. If you want a faster response - write me in slack.

**Pull request merged**